ccan/ccan/opt/opt.h

```diff
@@ -228,6 +228,15 @@ void opt_register_table(const struct opt_table *table, const char *desc);
 	_opt_register((names), OPT_CB_ARG((cb), OPT_EARLY, (show),(arg)), \
 		      (arg), (desc))
 
+/**
+ * opt_unregister - unregister an option.
+ * @names: the names it was registered with.
+ *
+ * This undoes opt_register[_early]_[no]arg.  Returns true if the option was
+ * found, otherwise false.
+ */
+bool opt_unregister(const char *names);
```

Comment:
BTW, CCAN updates should be done via "make update-ccan" (optional: CCAN_NEW="<new modules>".  Assumes that ccan you want is in ../ccan.

Easy to fix this post, however.

Reply:
I did it this way then withdrawn it because of the huge diff, will fix!